bokehjs/package-lock.json

```diff
@@ -1933,6 +1933,43 @@
       "integrity": "sha1-m387DeMr543CQBsXVzzK8Pb1nZQ=",
       "dev": true
     },
+    "patch-package": {
```

Comment:
Where this come from? It isn't in `package.json`.

Reply:
`npm install` needs to be run and changes to this file commited.

Reply:
ah missed that, it's cruft from the other version of the PR. I will fix it up in another small PR